apps/air-discount-scheme/backend/src/app/modules/user/user.service.ts

```diff
@@ -61,7 +66,7 @@ export class UserService {
             user.nationalId,
           )),
         ]
-        await this.cacheManager.set(cacheKey, { custodians }, { ttl: ONE_WEEK })
+        await this.cacheManager.set(cacheKey, { custodians }, ONE_WEEK * 1000)
```

Comment:
Maybe rename `ONE_WEEK ` to `ONE_WEEK_IN_MS` and update that constant with * 1000

Reply:
We need to stay backwards compatible with other ADS logic which uses a week in second, including API responses consumed by external systems.